### PR TITLE
add entitree-table and labels consts

### PR DESCRIPTION
**Pull request by mshd**

### CODE REVIEW

File: apps/entitree-images/package.json [hunk 1]
```diff
@@ -20,45 +20,36 @@
...
     "wikidata-sdk": "^8.0.0"
   },
   "devDependencies": {
-    "@types/jest": "^28.1.0",
-    "@types/react": "18.0.10",
+    "@types/react": "18.0.9",
     "eslint": "8.16.0",
     "eslint-config-next": "12.1.6",
     "jest": "^28.1.0",
-    "prisma": "^3.14.0",
```
Comment (ogroppo) [inline apps/entitree-images/package.json:61]: I think this is still needed, is it not better to have a separate PR for the other apps?

File: apps/entitree-images/lib/removePhotoBackground.ts [hunk 2]
```diff
@@ -5,51 +5,40 @@ import {
...
-  const streamPipeline = promisify(pipeline);
 
-  const formData = new FormData();
-  //Temp file needed for calling the API
-  const tmpFile = path.join(".", "uploads", "tmpFile" + Math.random());
-  await BUCKET.file("uploads/original/" + id + "").download({
-    destination: tmpFile,
-  });
-  //also supports image_file_base64, but couldn't figure out how to convert GS File to Base64
-  formData.append("image_file", fs.createReadStream(tmpFile));
```
Comment (mshd) [inline apps/entitree-images/lib/removePhotoBackground.ts:27]: finally fixed this @ogroppo

**Comment by mshd** [comment]
Yes you're right

File: apps/chordstop/components/ArtistSocialMedia.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { Artist } from "@prisma/client";
+import { Artist } from ".prisma/client";
```
Comment (ogroppo) [inline apps/chordstop/components/ArtistSocialMedia.tsx:1]: it should be @prisma/client... if it doesn't work than it means something has not been set up quite right

File: apps/chordstop/pages/api/auth/[...nextauth].ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import GoogleProvider from "next-auth/providers/google";
 import NextAuth from "next-auth";
 import { PrismaAdapter } from "@next-auth/prisma-adapter";
-import { PrismaClient } from "@prisma/client";
+import { PrismaClient } from ".prisma/client";
 
 const prisma = new PrismaClient();
```
Comment (ogroppo) [inline apps/chordstop/pages/api/auth/[...nextauth].ts:6]: this should be in a shared location, but I guess it not even used in chordstop?

File: apps/chordstop/components/ArtistSocialMedia.tsx (hunk 3 above)
Comment (mshd) [reply apps/chordstop/components/ArtistSocialMedia.tsx:1]: I changed it check the prisma schema so it doesn't conflict

Comment (ogroppo) [reply apps/chordstop/components/ArtistSocialMedia.tsx:1]: not sure what you mean, let's discuss... i know in some forums they say to use .prisma but I know for first-hand-exeprience it shouldn't be so

**Pull request merged**